plugins/catalog-react/src/components/EntityLifecyclePicker/EntityLifecyclePicker.tsx

```diff
@@ -47,7 +47,13 @@ const icon = <CheckBoxOutlineBlankIcon fontSize="small" />;
 const checkedIcon = <CheckBoxIcon fontSize="small" />;
 
 /** @public */
-export const EntityLifecyclePicker = () => {
+export type EntityLifecyclePickerProps = {
+  initialFilter?: string[];
+};
+
+/** @public */
+export const EntityLifecyclePicker = (props: EntityLifecyclePickerProps) => {
```

Comment:
Wonder if we can just define this type inline rather than having a separate type for this in the api-surface.

```suggestion
export const EntityLifecyclePicker = (props: {initialFilter?: string[]; }) => {
```

Reply:
thanks Ben! I updated it